CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 **Features**:
 
 - Provide Clang-CL support. ([#1161](https://github.com/getsentry/sentry-native/pull/1161), [crashpad#100](https://github.com/getsentry/crashpad/pull/100))
+- Enable Crashpad retry for non-iOS builds ([#1152](https://github.com/getsentry/sentry-native/pull/1152), [crashpad#120](https://github.com/getsentry/crashpad/pull/120))
```

Comment:
I would prefer to just state "enable crashpad report upload retry-mechansim for all platforms" rather than just non-iOS builds, which is technically correct, but won't mean a lot to most users not aware of the upstream build setup.